.taskcluster.yml

```diff
@@ -0,0 +1,26 @@
+version: 0
```

Comment:
`0`, not `1`?

Reply:
Sorry, I copied it from taskcluster-terraform.  I'l fix.